telemetry/definitions/commonDefinitions.json

```diff
@@ -3603,6 +3639,10 @@
                     "type": "codeTransformJobId",
                     "required": true
                 },
+                {
+                    "type": "codeTransformNumberOfTransformedFiles",
+                    "required": false
+                },
```

Comment:
I think we should keep this metric's purpose clear: an user tries to initiate a transformation either through /transform or some other channel. codeTransformNumberOfTransformedFiles would belong more in codeTransform_jobStart per visual studio's usage. In addition, when user actually submits a transformation, there should be metric on the service side that captures the lines of code, number of files, etc, so consider using those metrics instead of introducing new ones on the IDE side.

Reply:
This has been addressed in a more recent commit. We now emit this field from the `codeTransform_jobStart` metric type instead.